lighthouse-core/lib/emulation.js

```diff
@@ -41,10 +41,13 @@ const NEXUS5X_USERAGENT = {
     '(KHTML, like Gecko) Chrome/59.0.3033.0 Mobile Safari/537.36'
 };
 
+const LATENCY_FACTOR = 3.75;
+const THROUGHPUT_FACTOR = 1.1;
```

Comment:
+1.

it's equal to `throughput * 90/99` but you might as well simplify it to `throughput * 0.9`